go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/Venafi/vcert/v4
 
 require (
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
The latest version of this library is the 4.0.0 and it was released on May 2020 so this library is not updated since that time, so we need to evaluate if the benefit to use is worth the possible risks to use an old library.

Reply:
@marcos-albornoz The library has been removed and a regex validation for the system version was added instead